domains/proximity.json

```diff
@@ -0,0 +1,9 @@
+{
+  "description": "Dev Env",
+  "repo": "https://github.com/hexed",
+  "owner": {
+    "username": "hexed",
+    "email": "me@proximity.sh"
+  },
+  "record": {"A": "144.172.67.6"}
```

Comment:
A records have to be an array of ips

Reply:
Why an array ?  Only need the one up

Reply:
The schema as described in the docs requires that the value be an array since technically, "A" could have multiple values. Please change wrap the ip in an array and I'll merge it in.

Reply:
@phenax Got it.